apps/srt-file-transmit.cpp

```diff
@@ -42,14 +42,27 @@ written by
 #endif
 
 
-bool Upload(UriParser& srt, UriParser& file);
-bool Download(UriParser& srt, UriParser& file);
+bool Upload(UriParser& srt, UriParser& file, std::ostream &out_stats);
+bool Download(UriParser& srt, UriParser& file, std::ostream &out_stats);
 
 const srt_logging::LogFA SRT_LOGFA_APP = 10;
 
 static size_t g_buffer_size = 1456;
 static bool g_skip_flushing = false;
 
+map<string, string> g_options;
```

Comment:
Why did you try to copy the old `Option` functionality from stransmit? It's going to be abandoned and in some further development branches are already deleted. Please use the new one.

Reply:
Moved to the new one.